src/pages/events/index.svelte

```diff
@@ -96,19 +170,27 @@ li {
                 </div>
 
                 {#if event.more_info_url}
-                  <a href="{event.more_info_url}" target="_blank">
+                  <a href={event.more_info_url} target="_blank">
                     <Icon icon={faExternalLinkAlt} />
                     <small class="align-bottom">Event Website</small>
                   </a>
                 {/if}
               </div>
               <div class="col-auto align-self-start">
                 <div class="event-buttons-container">
-                  <a href="/requests?event={ encodeURIComponent(event.id) }" class="btn btn-primary d-block m-2">View Requests</a>
-                  {#if isParticipant($me, event) }
-                    <button class="btn btn-light d-block m-2" disabled>Added</button>
+                  <a
+                    href="/requests?event={encodeURIComponent(event.id)}"
+                    class="btn btn-primary d-block m-2">View Requests</a
+                  >
+                  {#if isParticipant($me, event)}
+                    <button class="btn btn-light d-block m-2" disabled
+                      >Added</button
```

Comment:
is this a formatting convention having the closing > here? I think I like
`<button>click</button>` or
`<button>
    click
</button>`
better. @Schparky?

Reply:
Agreed. I think it looks strange to have the `>` on the next line.